.gitignore

```diff
@@ -1,3 +1,4 @@
 *.gem
 Gemfile.lock
 .bundle
+*.swp
```

Comment:
I'm often working with vi and I would like to prevent from committing some swp files.